doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be usable by a novice who has never use commands application (commands that are simple and easy to understand)
```

Comment:
This seems to be a FR instead. The main thing to take note when writing NFR is that it should not reference features of your program.